src/main/java/mat/client/login/SecurityBannerModal.java

```diff
@@ -68,7 +69,7 @@ public SecurityBannerModal() {
         ModalHeader modalHeader = new ModalHeader();
         Div headerDiv = new Div();
         headerDiv.add(new HTML("System Use Notification"));
-        headerDiv.getElement().getStyle().setProperty("font-size","small");
+        headerDiv.getElement().getStyle().setProperty("fontSize", "small");
```

Comment:
Gotta love gwt. Some of the properties that they define you have to use camel case. For ones they don't its css name., font-size -> fontSize. Good catch.